SECURITY.md

```diff
@@ -0,0 +1,8 @@
+We acknowledge that every line of code that we write may potentially contain security issues.
+We are trying to deal with it responsibly and provide patches as quickly as possible.
+
+We host our bug bounty program on HackerOne, it is currently private, therefore if you would like to report a vulnerability and get rewarded for it, please ask to join our program by filling this form:
```

Comment:
s/HackerOne, it/HackerOne. It`

Reply:
Alternatively you can also make the word `form` to be the link.

Reply:
Can you address all change request here: https://github.com/zalando-incubator/new-project/blob/master/SECURITY.md.

Once it is agreed there we can update here as well. This PR is simply following what is proposed by the sample.

Reply:
That being said I still think we should merge this one and create a new PR to update this Repo once the "master" template good changed accordingly. @cberg-zalando what do you think?

Reply:
Sure. I was not aware that this was based on a project template.